src/Attachment.tsx

```diff
@@ -310,6 +311,8 @@ export const AttachmentView = (props: {
 
         default:
             var unknownAttachment = props.attachment as UnknownMedia;
+            if (unknownAttachment.thumbnailUrl)
+                return <Unknown format={ props.format } contentType={ unknownAttachment.contentType } contentUrl={ unknownAttachment.contentUrl } name={ unknownAttachment.name } thumbnailUrl={ unknownAttachment.thumbnailUrl } />
```

Comment:
Do you need this branch at all? Can't you just always return that first version, which will do the right thing if unknownAttachment.thumbnailUrl is undefined?